lib/smart_city/ingestion/transformation.ex

```diff
@@ -0,0 +1,54 @@
+defmodule SmartCity.Ingestion.Transformation do
```

Comment:
Does this module need any separate unit tests?

Reply:
We added a few tests to the Ingestion tests ¯\_(ツ)_/¯ It's honestly basic enough that I can't imagine it would need any - it's really just a complex way of defining an enum.
But you tell me, haha